packages/react-components/src/components/VideoGallery/Layout.ts

```diff
@@ -28,7 +28,7 @@ export interface LayoutProps {
   screenShareComponent?: JSX.Element;
   /**
    * Maximum number of participant remote video streams that is rendered.
-   * @defaultValue 4
+   * @defaultValue 9
```

Comment:
@PorterNan would this be considered a breaking change? Should we leave the default as `4` in the VideoGallery, but at the composite level choose 9?